spyder/plugins/editor/widgets/tests/test_codeeditor.py

```diff
@@ -264,5 +270,96 @@ def test_undo_return(editorbot):
     assert editor.toPlainText() == text
 
 
+def test_brace_match():
+    """Tests for the highlighting of matching parenthesis, braces and brackets
+
+    Specifically provides regresion tests for issues
+     * spyder-ide/spyder#2965
+     * spyder-ide/spyder#9179
+     * spyder-ide/spyder#14374
+
+    If this test fails the best way to investigate is probably to open
+    assets/braces.py in Spyder, step through the file and visually
+    observe brace matching.
+
+    Some caveats for brace matching can be found in pull request
+    spyder-ide/spyder#14376
+
+    The functions being tested are essentially:
+     * TextEditBaseWidget.find_brace_match
+     * CodeEditor.in_comment
+     * CodeEditor.in_string
+    """
+    # Create editor with contents from assets/brackets.py
+    app = qapplication()
+    editor = codeeditor.CodeEditor(parent=None)
+    editor.setup_editor(language='Python')
```

Comment:
This is not necessary. Instead, you need to pass as an argument to `test_brace_match` the fixture called `editorbot`. That's in charge is creating and setting up a CodeEditor instance for you.

With that, at this point you only need to write:

    __, editor = editorbot

to get a reference to the instance created by `editorbot` and use it below.